device-types/Datacom/DM4610-OLT-4GPON+4GX+2XS.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Datacom
+model: DM4610 OLT 4GPON+4GX+2XS
+slug: dm4610-olt-4gpon+4gx+2xs
+part_number: 800.5285.xx
```

Comment:
According to [this datasheet](https://www.datacom.com.br/uploads/product/file/a4d6b18b9b0cc9b4ac5dccfa22678fdd.pdf) part number `800.5285.xx` which you have on the `DM4610 OLT 4GPON+4GX+2XS` actually belongs to the `DM4612 OLT 8GPON+2GT+2XS`. Can you please correct this? Or can you provide the datasheet with this part number listed? If you do not have the actual part number, please remove this definition

Reply:
ok, removed.

Reply:
Looks like the part number is still here, I am unresolving this so you remember to remove/update it

Reply:
Sorry.
Adjusted.